.github/workflows/release.yml

```diff
@@ -109,9 +116,23 @@ jobs:
           # fi
 
           if [[ ${input_holochain_source_branch} != "" ]]; then
-            echo "::set-output name=holochain_source_branch::${input_holochain_source_branch}"
+            export holochain_source_branch="${input_holochain_source_branch}"
+          else
+            export holochain_source_branch="${GITHUB_HEAD_REF:-${GITHUB_REF_NAME}}"
+          fi
+          echo "::set-output name=holochain_source_branch::${holochain_source_branch}"
+
+          if [[ ${input_holochain_target_branch} != "" ]]; then
+            echo "::set-output name=holochain_target_branch::${input_holochain_target_branch}"
           else
-            echo "::set-output name=holochain_source_branch::${GITHUB_HEAD_REF:-${GITHUB_REF_NAME}}"
+            case "${holochain_source_branch}" in
+              develop*)
+                echo "::set-output name=holochain_target_branch::$(echo ${holochain_source_branch} | sed 's/^develop/main/')"
+                ;;
+              *)
+                echo "::set-output name=holochain_target_branch::main"
```

Comment:
o_O - is this a dangerous fallback? Do we ever want to release from some randomly named branch into main? Or did I misread this?

Reply:
thanks for the careful observation.

since we fully define the release procedure we can just enable dry-run mode from non-develop prefixed branches.